azure-pipelines/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
 - job: macOS
   pool:
-    vmImage: macOS 10.13
+    vmImage: macOS 10.15
```

Comment:
You need to add a hyphen, [like this](https://github.com/AArnott/Library.Template/commit/ba764031a80c365dbf0182c67fda295fd89aaa5b#diff-6f48d6e0be33ef564632a0a577eb3504).

Reply:
Good catch! Thanks